tests/core/pyspec/eth2spec/test/electra/epoch_processing/test_process_pending_consolidations.py

```diff
@@ -302,3 +302,67 @@ def test_pending_consolidation_with_pending_deposit(spec, state):
     # Pending deposit to the source was not processed.
     # It should only be processed in the next epoch transition
     assert state.pending_deposits == [pending_deposit]
+
+
+@with_electra_and_later
+@spec_state_test
+def test_pending_consolidation_source_balance_less_than_effective(spec, state):
```

Comment:
Actually, what matters for the test scenario is that `balance[i] < validators[i].effective_balance`, so the test name specifies it precisely. For this scenario the effective balance can be anything in `[MIN_ACTIVATION_BALANCE, MAX_EFFECTIVE_BALANCE_ELECTRA]` range

Reply:
Idea for another couple of tests: since these tests are around `max_effective_balance` for the source, should we also have tests with compounding creds validators that have higher max effective balance?